test/integration/registerable_test.rb

```diff
@@ -124,7 +124,9 @@ def user_sign_up
     #    https://github.com/mongoid/mongoid/issues/756
     (pending "Fails on Mongoid < 2.1"; break) if defined?(Mongoid) && Mongoid::VERSION.to_f < 2.1
 
-    create_user
+    user = create_user
+    user.update_attribute(:username, nil)
```

Comment:
Would mind to explain why this change was needed?

Reply:
I don't remember now...I'll pick this back up in the next few days along with the other comments you made.